veros/cli/veros_run.py

```diff
@@ -57,6 +57,7 @@ def run(setup_file, *args, **kwargs):
         "float_type",
         "diskless_mode",
         "force_overwrite",
+        "linear_solver",
```

Comment:
Same here, this doesn't really need to be exposed. Regular users shouldn't touch this setting.

Reply:
Yeah I agree with this